scrapers/Spacebattles/readme.md

```diff
@@ -0,0 +1,17 @@
+Navigate to the "noveltorpedo/scrapers/Spacebattles" directory
+before proceeding.
+
+Install python dependencies:
+```
+pip install -r requirements.txt
+```
+
+Start up elasticsearch and database. (instructions in the website readme)
```

Comment:
You can put an actual (relative) link here; see the main page readme for an example.